### PR TITLE
Improvements Targeting Windows

### PR DESCRIPTION
**Pull request by DDEle**
## Type of Change: features
API not changed

## Description
- Avoid most of warnings when compiling with MSVC (https://github.com/intel/neural-speed/actions/runs/7998735186/job/21845460790#step:3:959)
- Resolve lto warnings on Linux
- Include missing pyd files
- PPL supports readding cnn_dailymail
- Using /utf-8 on Windows

(actually they are non-mha changes on the mha-avx2 branch)

## Dependency Change?
No

### CODE REVIEW

**Review by zhewang1-intc — APPROVED** [review]
LGTM

File: CMakeLists.txt [hunk 1]
```diff
@@ -96,9 +96,28 @@ endif()
...
+    # MultiByteToWideChar() and WideCharToMultiByte() from <Windows.h> instead. You can define
+    # _SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING or _SILENCE_ALL_CXX17_DEPRECATION_WARNINGS to suppress this
+    # warning.
+    add_compile_definitions(_SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING)
+
+    # Microsoft renamed some POSIX and Microsoft-specific library functions in the CRT to conform with C99 and C++03
+    # constraints on reserved and global implementation-defined names. If you need to use the existing function names
+    # for portability reasons, you can turn off these warnings. The functions are still available in the library under
+    # their original names.
+    add_compile_definitions(_CRT_NONSTDC_NO_WARNINGS)
```
Comment (airMeng) [inline CMakeLists.txt:120]: put them under ```cmake/Common.cmake```

**Review by airMeng — COMMENTED** [review]
LGTM except minor comments

**Review by zhentaoyu — APPROVED** [review]

**Comment by airMeng** [comment]
after https://github.com/intel/neural-speed/pull/135 merged, you can rebase and trigger CI again

File: CMakeLists.txt (hunk 1 above)
Comment (DDEle) [reply CMakeLists.txt]: Updated.

You can compare https://github.com/intel/neural-speed/compare/ddd901dd83e00c5ada93cc507b36759f28c87da1..27417bc6f92a6c577790b23e21c79f7fd820cfff

**Pull request merged**